linchpin/FilterUtils/FilterUtils.py

```diff
@@ -293,3 +293,10 @@ def write_to_file(data, path, filename):
     fd.write(json.dumps(data))
     fd.close()
     return data
+
+
+def convert_to_json(input_str):
```

Comment:
@samvarankashyap

Reply:
@14rcole I think it is a problem with ansible. 
Am not sure why rundb is returning string instead of json. 
I remember you fixed that.
but it came back again.